<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>list</class>
 <widget class="QWidget" name="list">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="elementLineEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="maxLength">
            <number>240</number>
           </property>
           <property name="placeholderText">
            <string>add a new item to the list</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addElementButton">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="doneButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mark &lt;span style=&quot; font-weight:600;&quot;&gt;done&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:7pt; font-style:italic; color:#999999;&quot;&gt;Instead of this &lt;/span&gt;&lt;span style=&quot; font-size:7pt; font-weight:600; font-style:italic; color:#999999;&quot;&gt;click twice&lt;/span&gt;&lt;span style=&quot; font-size:7pt; font-style:italic; color:#999999;&quot;&gt; on element with &lt;/span&gt;&lt;span style=&quot; font-size:7pt; font-weight:600; font-style:italic; color:#999999;&quot;&gt;right mouse button&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/graphics/icons/done.png</normaloff>:/graphics/icons/done.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>60</width>
               <height>120</height>
              </size>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change to &lt;span style=&quot; font-weight:600;&quot;&gt;not done&lt;/span&gt; or &lt;span style=&quot; font-weight:600;&quot;&gt;delete&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/graphics/icons/delete.png</normaloff>:/graphics/icons/delete.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>60</width>
               <height>120</height>
              </size>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
